Cargo.toml

```diff
@@ -40,4 +40,11 @@ members = [
 ]
 
 [profile.release]
+# Polkadot runtime requires unwinding.
```

Comment:
```suggestion
```

Reply:
I kept it so we know **why** but it does not matter much.